pkg/image/apiserver/admission/apis/imagepolicy/v1/defaults.go

```diff
@@ -24,6 +24,7 @@ func SetDefaults_ImagePolicyConfig(obj *ImagePolicyConfig) {
 		obj.ResolutionRules = []ImageResolutionPolicyRule{
 			{TargetResource: GroupResource{Resource: "pods"}, LocalNames: true},
 			{TargetResource: GroupResource{Group: "build.openshift.io", Resource: "builds"}, LocalNames: true},
+			{TargetResource: GroupResource{Group: "", Resource: "builds"}, LocalNames: true},
```

Comment:
this group no longer exists.  oapi is dead

Reply:
when i was testing this was definitely how the admission plugin was seeing the resources i was creating... until i added this, it was ignoring builds... and i was creating the build as:
```
apiVersion: build.openshift.io/v1
kind: Build
metadata:
  name: testbuild2
```

So are you sure we don't still have something somewhere converting things into legacy?

Reply:
```
I0220 23:52:39.898549       1 imagepolicy.go:244] looking at schemagr: schema.GroupResource{Group:"", Resource:"builds"}
I0220 23:52:39.898571       1 imagepolicy.go:245] looking at apigr: v1.GroupResource{Group:"", Resource:"builds"}
```

when i added
```
	schemagr := attr.GetResource().GroupResource()
	apigr := metav1.GroupResource{Resource: schemagr.Resource, Group: schemagr.Group}

	glog.Infof("looking at schemagr: %#v", schemagr)
	glog.Infof("looking at apigr: %#v", apigr)
```

to the admi() function.

So we've got a disconnect somewhere...

Reply:
opened https://bugzilla.redhat.com/show_bug.cgi?id=1679602

Reply:
@deads2k are you ok w/ merging with this in place, pending resolution of the bug?  I can put a TODO in to remove it when the bug is resolved.